src/backend/utils/graphics/vulkan/index.ts

```diff
@@ -1,22 +1,26 @@
 import { vulkanHelperBin } from 'backend/constants'
-import { spawnSync } from 'child_process'
+import { SpawnSyncReturns, spawnSync } from 'child_process'
 import { gte as semverGte } from 'semver'
 
 type VulkanVersion = [maj: number, min: number, patch: number]
 
+let instance_version: SpawnSyncReturns<string> | null = null
+let physical_versions: SpawnSyncReturns<string> | null = null
```

Comment:
Could use something like
```suggestion
let instance_version: ReturnType<typeof spawnSync> | null = null
let physical_versions: ReturnType<typeof spawnSync> | null = null
```
here to always have the correct type & avoid the import